packages/mdctl-axon-tools/__tests__/TOOLS-50/TOOLS-50.test.js

```diff
@@ -26,14 +26,14 @@ describe('getStudyManifestEntities', () => {
     ['c_sites'],
     ['c_anchor_date_templates'],
     ['c_looker_integration_records'],
-    ['int__vendor_integration_records'],
```

Comment:
@mmulgonkar-pr Hey I've see you've removed a bunch of objects. Are you confident that those object don't exist anywhere and will never have to be exported? They may not be in you're codebase any more but is it possible they're in use somewhere?

Reply:
@fiachra those were the objects used in the Integration platform codebase. But during recent refactoring, we have removed those objects and added some new objects.

Reply:
@mmulgonkar-pr No problem, I figured that. What I 'm asking is about backward compatibility.  Are you 100% confidence that there are no examples of studies in the wild using the older version of your platform. If there are, then you'll be breaking the ability to migrate those objects?

Reply:
@fiachra got it, so you would suggest keeping the old objects as is and adding the new ones ?
I will make that change right away. Thanks